### ISSUE TITLE
Absent platelet dense granules

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6421]
**Preferred term label:**
https://hpo.jax.org/app/browse/term/HP:0012530

**Synonyms**
Absent platelet dense bodies

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
608233 
PMID:11809908  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also, Analysis of PFA-fixed platelets loaded with calcium by electron microscopy showed a large reduction in the number of dense granules (mean numbers of 1 per patient platelet versus 3.55 per control platelet 
same disease
https://pubmed.ncbi.nlm.nih.gov/16551969/

**Comment by pnrobinson** [comment]
Also abnormal size of dense granules, see Figure 7 of https://pubmed.ncbi.nlm.nih.gov/16551969/

**Comment by pnrobinson** [comment]
done
